pkg/recipes/terraform/config/config.go

```
@@ -38,10 +44,12 @@ const (
 
 // New creates TerraformConfig with the given module name and its inputs (module source, version, parameters)
 // Parameters are populated from environment recipe and resource recipe metadata.
-func New(moduleName string, envRecipe *recipes.EnvironmentDefinition, resourceRecipe *recipes.ResourceMetadata) *TerraformConfig {
+func New(ctx context.Context, moduleName string, envRecipe *recipes.EnvironmentDefinition, resourceRecipe *recipes.ResourceMetadata, envConfig *recipes.Configuration, armOptions *arm.ClientOptions) *TerraformConfig {
+	// Getting the module source path with credentials information if its a private source.
+	path, _ := getModuleSource(ctx, envConfig, envRecipe.TemplatePath, armOptions)
```

Are we ignoring the error here?

---

good catch! i handled it my latest pr